builtin/providers/openstack/resource_openstack_compute_secgroup_v2.go

```diff
@@ -210,10 +212,20 @@ func resourceComputeSecGroupV2Delete(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error creating OpenStack compute client: %s", err)
```

Comment:
Actually, "creating" is correct here - it's making a new OpenStack client connection in order to do the delete. :smile: 

However, I do have a slightly cleaned up version coming right up.